src/api/postProjectData.ts

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import { url } from '../shared/variable.shared';
+import { useUserStore } from 'src/stores/user';
+
+const createProject = async (project: IProjectCreate): Promise<number> => {
+  const userStore = useUserStore();
+  const teamID =
+    userStore.getUser.email &&
+    (await userStore.loadTeam(userStore.getUser.email));
+
+  const res = { ...project, teamID };
+
+  try {
+    const { data } = await axios.post(`${url}/projects`, res, {
```

Comment:
tutaj to samo co wyżej tylko zwracany jest projekt, można zrobić jakieś dto dla projektu

Reply:
Można kiedyś zrobić taska na clean code i dodać takiego readera